tests/json-fixtures/test_transactions.py

```diff
@@ -40,68 +41,92 @@
 BASE_FIXTURE_PATH = os.path.join(ROOT_PROJECT_DIR, 'fixtures', 'TransactionTests')
 
 
+# Fixtures have an `_info` key at their root which we need to skip over.
+FIXTURE_FORK_SKIPS = {'_info', 'rlp'}
```

Comment:
@cburgdorf the `_info` keys makes sense.  Can you fill me in on why the `'rlp'` key is here?

Reply:
The forks are on the same level in the fixtures as `_info` and `rlp` so I'm just iterating over all keys and assume everything but `_info` and `rlp` is holding fork specific data.

```
{
    "TransactionWithGasLimitOverflow63_1" : {
        "Byzantium" : {
            "hash" : "2a9ef120d86d735b6b7b2a43462d2567a4e1605d18fa5b3c70cb956cbcb54e3a",
            "sender" : "cda94a77b8357ff651db2e3828216440c4b732e8"
        },
        "Constantinople" : {
            "hash" : "2a9ef120d86d735b6b7b2a43462d2567a4e1605d18fa5b3c70cb956cbcb54e3a",
            "sender" : "cda94a77b8357ff651db2e3828216440c4b732e8"
        },
        "EIP150" : {
            "hash" : "2a9ef120d86d735b6b7b2a43462d2567a4e1605d18fa5b3c70cb956cbcb54e3a",
            "sender" : "cda94a77b8357ff651db2e3828216440c4b732e8"
        },
        "EIP158" : {
            "hash" : "2a9ef120d86d735b6b7b2a43462d2567a4e1605d18fa5b3c70cb956cbcb54e3a",
            "sender" : "cda94a77b8357ff651db2e3828216440c4b732e8"
        },
        "Frontier" : {
            "hash" : "2a9ef120d86d735b6b7b2a43462d2567a4e1605d18fa5b3c70cb956cbcb54e3a",
            "sender" : "cda94a77b8357ff651db2e3828216440c4b732e8"
        },
        "Homestead" : {
            "hash" : "2a9ef120d86d735b6b7b2a43462d2567a4e1605d18fa5b3c70cb956cbcb54e3a",
            "sender" : "cda94a77b8357ff651db2e3828216440c4b732e8"
        },
        "_info" : {
            "comment" : "",
            "filledwith" : "cpp-1.3.0+commit.1829957d.Linux.g++",
            "lllcversion" : "Version: 0.4.18-develop.2017.10.11+commit.81f9f86c.Linux.g++",
            "source" : "src/TransactionTestsFiller/ttGasLimit/TransactionWithGasLimitOverflow63_1Filler.json",
            "sourceHash" : "7673d796a4a614c533ac61ecb0f8c49158e06d8b86932813afd3d0be13ae56e4"
        },
        "rlp" : "0xf865800188800000000000000194095e7baea6a6c7c4c2dfeb977efac326af552d870b801ba048b55bfa915ac795c431978d8a6a992b628d557da5ff759b307d495a36649353a01fffd310ac743f371de3b9f7f9cb56c0b28ad43601b4ab949f53faa07bd2c804"
    }
}
```

Reply:
As an alternative we could also do a second check to see if the key is actually an object or a primitive? I just went for a basic blacklist right away ;)

Reply:
Lets change this to use the following logic.

- if the key is `_info` or `rlp`, skip
- if the key is in the list of known fork names, yield
- else throw error.

This will guarantee we don't silently skip something at a later date if the format changes.

Reply:
We basically do that already because any key that is not in the list of know fork names will throw [here](https://github.com/cburgdorf/py-evm/blob/e8e86e5bbb5ba7aca03d5f3cfbd4c08bf14f3537/tests/json-fixtures/test_transactions.py#L102)